int32 seqence_number
time outgoing_stamp
time pong_stamp
time received_stamp
